.github/scripts/configure-proxy.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# For running the script, see go/templates-gitactions-script
```

Comment:
I think this link is wrong - could we just link to the contributor-docs callout?

Reply:
It would be good to add similar links in the other scripts

Reply:
Good catch, I'll change that and do the same for the others